.travis.yml

```diff
@@ -21,22 +21,25 @@ matrix:
     - php: 5.6
       env: SYMFONY_VERSION=2.3.*
     - php: 5.6
-      env: SYMFONY_VERSION=2.5.*
-    - php: 5.6
       env: SYMFONY_VERSION=2.7.*
     - php: 5.6
       env: SYMFONY_VERSION=3.0.*
+    - php: 5.3.3
+      env: SYMFONY_VERSION="2.3.*,>=2.3.4" COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
should the >=2.3.4 be in the composer.json? maybe in the form of conflicts: symfony < 2.3.4?